Functions.Templates/Templates/DurableFunctionsActivity-Python-1.x/metadata.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "defaultFunctionName": "Hello",
+  "description": "$DurableFunctionsActivity_description",
+  "name": "Durable Functions activity (upgrade to bundles V2)",
```

Comment:
I would lead towards removing this.

I'm fine with leaving out any nudging towards V2 for now, so that when we do introduce it, it is a clean way.

Reply:
Alrighty then, removed!